includes/class-wc-order.php

```diff
@@ -1414,6 +1414,12 @@ public function get_downloadable_items() {
 				continue;
 			}
 
+			// check item refunds
```

Comment:
Just need to fix coding standards for this comment, need to start with capital letter and end with period.
Like `// Check item refunds.`
Thanks, just this minor issue to check.

Reply:
@claudiosanches fixed coding standards for this comment.